example/code_java_maven/src/main/java/app/Logger.java

```diff
@@ -0,0 +1,23 @@
+package app;
+
+public class Logger {
+
+    public static void logCurrentTest(int expected, int received, int a, int b) {
+        System.err.println("> calculator(" + a + " and " + b + ") = " + received);
```

Comment:
C'est possible d'être plus explicite, actuellement pour chaque opération (+, -, *, %) c'est `and` qui est affiché, tu peux sûrement adapter l'affichage pour chaque opération :
add(2, 3) = 0
expected = 5
etc..

Reply:
done